app/serializers/base_serializer.rb

```diff
@@ -123,6 +122,18 @@ def show_policy?
     return respond_to_policy && respond_to_manage && can_manage
   end
 
+  def determine_submitter(object)
+    # FIXME: needs to be the creators for assets
+    return object.owner if object.respond_to?('owner')
```

Comment:
I'm not sure this is needed, everything should have a contributor rather than owner, of if not have an alias or be updated

Reply:
This is a more general serialization issue and IMO should be raised as one